packages/oc-template-react-compiler/lib/viewTemplate.js

```diff
@@ -8,7 +8,9 @@ const viewTemplate = ({
   var modelHTML =  model.__html ? model.__html : '';
   var staticPath = model.reactComponent.props._staticPath;
   var props = JSON.stringify(model.reactComponent.props);
-  return '<div id="${reactRoot}" class="${reactRoot}">' + modelHTML + '</div>' +
+  var randomId = Math.random() * 10000000;
+  var reactRoot = "${reactRoot}" + '-' + randomId;
+  return '<div id="'+ reactRoot +'" class="'+ reactRoot +'">' + modelHTML + '</div>' +
```

Comment:
^ follows previous suggestion
```suggestion
  return '<div id="'+ reactRootId +'" class="${reactRoot}">' + modelHTML + '</div>' +
```